### ISSUE TITLE
zombified piglins // pigzombie

### ISSUE THREAD

**Issue by voigtman** [MLG-Fortress/ExtraHardMode#238]
Zombified Piglins are not aggressive despite the config calling for Pig Zombies to be always aggressive...

**Comment by RoboMWM** [comment]
Afaik the PigZombie class still exists without deprecation, and there is no "Zombie Piglin" class so I assume that same class is what controls the piglins. So... if anger works differently then I guess it's a matter of finding out what that is...

**Comment by voigtman** [comment]
The official namespace JE name for the new mob is zombified_piglin
[https://minecraft.gamepedia.com/Zombified_Piglin](url)

and the Bukkit API specifies 
public static final EntityType ZOMBIFIED_PIGLIN
[https://hub.spigotmc.org/javadocs/spigot/org/bukkit/entity/EntityType.html#ZOMBIFIED_PIGLIN](url)

**Comment by RoboMWM** [comment]
yes EntityType changed but there is no such class, so I assume it still uses the PigZombie class - and I have yet to receive any reports about an invalid cast.

**Comment by RoboMWM** [comment]
Has aggression behavior changed? I don't remember reading anything about it so I assume no...

**Comment by 3Lord3** [comment]
There is still a problem. Zombies piglins are not aggressive, although this feature is enabled in the plugin config.
